docs/assemblies/contributing.adoc

```diff
@@ -110,29 +110,69 @@ directory in dataplane-operator.
 
 kuttl tests are under the
 https://github.com/openstack-k8s-operators/dataplane-operator/tree/main/tests/kuttl/tests['tests/kuttl/tests']
-in dataplane-operator.
+in dataplane-operator and can be executed either directly from dataplane operator repository, or from https://github.com/openstack-k8s-operators/install_yamls[install_yamls]
 
 ==== Requirements
 
 The kuttl tests require a running cluster with
 https://github.com/openstack-k8s-operators/openstack-ansibleee-operator[openstack-ansibleee-operator], https://github.com/openstack-k8s-operators/openstack-baremetal-operator[openstack-baremetal-operator] and https://github.com/openstack-k8s-operators/infra-operator[infra-operator]
-running in the cluster.
+running in the cluster. The easiest way to ensure presence of these operators is to use install_yamls. However, it is possible to deploy them manually as well.
 
 ==== From install_yamls
 
-The kuttl tests are run from the
+First clone both the dataplane-operator and the install-yamls repos and make sure that they are in a state (commit) you want to test.
+For the purposes of this guide, we will assume following directory layout:
+
+....
+~
+├── dataplane-operator
+└── install_yamls
+....
+
+[NOTE]
+====
+The kuttl tests are also run from the
 https://github.com/openstack-k8s-operators/install_yamls[install_yamls]
 repository by the CI jobs.
+====
 
-Running from `install_yamls`:
+First initialize CRC. Make sure to have your pull secret ready and to fulfill all HW requirements. 
+CRC can be initialized without install_yamls utilities. However, using existing scripts will simplify debugging and issue reporting.
 
 [,sh]
 ----
-cd install_yamls
-# Set environment variables if needed to use a specific repo and branch of dataplane-operator
-export DATAPLANE_REPO=https://github.com/openstack-k8s-operators/dataplane-operator.git
-export DATAPLANE_BRANCH=main
-make dataplane_kuttl
+cd install_yamls/devsetup
+CPUS=12 MEMORY=25600 DISK=100 make crc
+----
+
+Prepare baremetal operator, ensure presence of other dataplane operator requirements and cleanup all remains of possible previous kuttl test run.
+
+[,sh]
+----
+cd ..
+make crc_bmo_setup
```

Comment:
this shouldn't be needed anymore since https://github.com/openstack-k8s-operators/install_yamls/pull/740 merged

Reply:
so just `baremetal` now. Got it.